sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -191,20 +191,22 @@ my_input = [
 
 See the service documentation for all [supported languages][supported_languages].
 
-### Return value
+### Long-Running Operations
+Long-running operations are operations which consist of an initial request sent to the service to start an operation,
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
+succeeded, to get the result.
 
-There are primarily two types of return values when checking on the result of a translation job - `JobStatusResult` and `DocumentStatusResult`.
-* A `JobStatusResult` will contain the details of the entire job, such as it's status, ID, any errors, and status summaries of the documents in the job.
-* A `DocumentStatusResult` will contain the details of an individual document, such as it's status, translation progress, any errors,
-and the URLs to the source document and translated document.
+Methods that translate documents are modeled as long-running operations.
+The client exposes a `begin_<method-name>` method that returns an `DocumentTranslationPoller` or `AsyncDocumentTranslationPoller`. Callers should wait
+for the operation to complete by calling `result()` on the poller object returned from the `begin_<method-name>` method.
+Sample code snippets are provided to illustrate using long-running operations [below](#examples "Examples").
 
 ## Examples
 
 The following section provides several code snippets covering some of the most common Document Translation tasks, including:
 
 * [Translate your documents](#translate-your-documents "Translate Your Documents")
-* [Check status on individual documents](#check-status-on-individual-documents "Check Status On Individual Documents")
```

Comment:
is the reason to hide this from the Readme because of some user feedback or service feedback?

Reply:
Was actually aligning with the .NET README on this one. This sample became kind of redundant since it's almost identical to the first example now that we return the pageable of DocumentStatusResult upon completion

Reply:
lol oh those .NET people. I should have reviewed what I did lol
I agree with you on not showing it. :)